README.md

```diff
@@ -97,6 +97,7 @@ Hertz is distributed under the [Apache License, version 2.0](https://github.com/
 - Email: [conduct@cloudwego.io](conduct@cloudwego.io)
 - How to become a member: [COMMUNITY MEMBERSHIP](https://github.com/cloudwego/community/blob/main/COMMUNITY_MEMBERSHIP.md)
 - Issues: [Issues](https://github.com/cloudwego/hertz/issues)
+- Slack: Join our CloudWeGo community [Slack Channel](https://join.slack.com/t/cloudwego/shared_invite/zt-tmcbzewn-UjXMF3ZQsPhl7W3tEDZboA).
```

Comment:
I think readme_CN.md also needs to be added and the pr content needs to be supplemented with Chinese instructions according to the template

Reply:
> I think readme_CN.md also needs to be added and the pr content needs to be supplemented with Chinese instructions according to the template

done